src/jupyter_contrib_nbextensions/nbextensions/accessibility_toolbar/font_style/color_control.js

```diff
@@ -7,11 +7,18 @@ define(["base/js/namespace", "jquery", "./spectrum"], function(
 
   //=============
```

Comment:
Please remove all commented out code from the file

Reply:
Sure I will do :)
If the code works for you guys, I will solve the conflict

Reply:
Once you've solved conflicts ill double check all works then merge